packages/language-server/src/manifest-handling.ts

```diff
@@ -135,6 +131,8 @@ async function readManifestFile(
   const flexEnabled = manifestJsonObject["sap.ui5"]?.flexEnabled;
   const minUI5Version =
     manifestJsonObject["sap.ui5"]?.dependencies?.minUI5Version;
-
-  return { flexEnabled: flexEnabled, minUI5Version: minUI5Version };
+  return {
+    flexEnabled: flexEnabled,
+    minUI5Version: minUI5Version,
+  };
```

Comment:
Maybe revert if it is only a change in formatting?

Reply:
with current formatting and running ```format:validate```  shows every thing is compliant with Prettier code style.. So it should be fine,